Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rrplug = "4.0.0"
+# rrplug = "4.0.0"
+rrplug = "=4.1.0"
```

Comment:
out of curiosity, is there a reason you are locking to that specific version in `Cargo.toml` as well? ^^

Reply:
because I might not follow semver for rrplug :trollface:

Reply:
:dread: